build.gradle

```diff
@@ -70,6 +70,13 @@ configurations.all {
         force "org.apache.commons:commons-lang3:3.4"
         force "org.springframework:spring-core:5.3.20"
         force "com.google.guava:guava:30.0-jre"
+        force "com.fasterxml.woodstox:woodstox-core:6.4.0"
+        force "org.scala-lang:scala-library:2.13.9"
+        force "org.yaml:snakeyaml:1.32"
```

Comment:
Can you remove snakeyaml and netty from this section?

Reply:
Done